.golangci.yml

```diff
@@ -1,3 +1,4 @@
+version: "2"
```

Comment:
massaging this into the v2 format

Reply:
Did you run the command? Or convert manually?